fixed_frame: "world"
object_frame: "reach_object"
results_directory: "/home/mripperger/ros/robot_reach_study/src/10.22710/basf_support/reach_study/results/"
pcd_filename: "/home/mripperger/ros/robot_reach_study/src/10.22710/basf_support/point_clouds/ram_1500.pcd"
planning_group: "gantry_manipulator"
get_avg_neighbor_count: false
compare_dbs: []
visualize_results: true

optimization:
  radius: 0.4
  max_steps: 10
  step_improvement_threshold: 0.01

ik_solver_config:
  name: "reach_plugins/ik/DiscretizedMoveItIKSolver"
  distance_threshold: 0.0 #0.100
  discretization_angle: 0.785
  planning_group: "gantry_manipulator"
  collision_mesh_filename: "package://basf_support/meshes/basf_support/visual/ram_1500_with_windows.ply"
  collision_mesh_frame: "reach_object"
  touch_links: ["world"]
  evaluation_plugin:
    name: "reach_plugins/evaluation/MultiplicativeFactory"
    plugins:
      - name: "reach_plugins/evaluation/ManipulabilityMoveIt"
        planning_group: "manipulator"
      - name: "reach_plugins/evaluation/DistancePenaltyMoveIt"
        planning_group: "gantry_manipulator"
        distance_threshold: 0.100
        exponent: 2
        collision_mesh_filename: "package://basf_support/meshes/basf_support/visual/ram_1500_with_windows.ply"
        collision_mesh_frame: "reach_object"
        touch_links: ["world"]
      #- name: "reach_plugins/evaluation/JointPenaltyMoveIt"
        #planning_group: "manipulator"

display_config:
  name: "reach_plugins/display/MoveItReachDisplay"
  planning_group: "gantry_manipulator"
  collision_mesh_filename: "package://basf_support/meshes/basf_support/visual/ram_1500_with_windows.ply"
  collision_mesh_frame: "reach_object"
  fixed_frame: "world"
  marker_scale: 0.2
